patient-appointment-api.yml

```diff
@@ -101,16 +101,16 @@ paths:
       tags:
         - Scheduled Appointments
       summary: Patient Data Processing
-      description: "# Data Requirements\n\n**Populate all known data**. It is required\
+      description: "\n# Data Requirements\n\n**Populate all known data**. It is required\
```

Comment:
interesting that this and other descriptions need/have a newline char at the beginning